File: Implementations/PrimeCheck.pl
print "Enter a number\n";
my $num = <STDIN>;
for my $i (2..sqrt($num))
{
	if ($num % $i == 0)
	{
	    printf("%d is not prime: %d*%d\n", $num, $i, $num / $i);
	    exit;
	}
}
printf("%d is prime\n1", $num);

File: Implementations/PrimeCheck.lua
print("Enter a number")
local num = io.read()
for i = 2,math.sqrt(num),1 
do 
    if (num % i == 0) then
        print(string.format('%d is not prime: %d*%d', num, i, num/i))
        os.exit()
    end
end
print(string.format("%d is prime", num))